manipulate_pim_data/rule/general_information_on_rule_format.rst

```diff
@@ -551,6 +551,96 @@ The expected values are:
             - type: clear
               field: associations
 
+Calculate
+_________
+
+This action allows to calculate numeric attribute values, with simple mathematical operations (addition, substraction, multiplication and division)
+
+This action only accepts number, measurement or price collection attributes for both source and destination.
+
+The action is split in 3 distinct parts:
+
+**destination**: the value you want to update. It can be composed of :
+
+- field: the attribute code (required)
+- locale: locale code if the attribute is localizable (optional)
+- scope: channel code if the attribute is scopable (optional)
+- currency: currency code if the attribute is a price collection (optional)
+- unit: unit code if the attribute is a measurement (optional, the default measurement unit of the attribute will be used if not set)
+
+**source**: the first operand of the operation. It can be composed of either:
+
+- value: a constant value (number, for instance 3.14)
```

Comment:
(required ?)

Reply:
unclear about the 'or' here

Reply:
@camillebourjade it means that the `source` can be either:
```yaml
source:
  value: 3.14
```
or
```yaml
source:
  field: price
  locale: en_US
  scope: ecommerce
  currency: EUR
```
(you cannot have both `field` and `value` in the definition)